diff --git a/ipc/chromium/src/third_party/libevent/arc4random.c b/ipc/chromium/src/third_party/libevent/arc4random.c
index 8729f6b92f..6f91f8df42 100644
--- a/ipc/chromium/src/third_party/libevent/arc4random.c
+++ b/ipc/chromium/src/third_party/libevent/arc4random.c
@@ -484,7 +484,8 @@ arc4random(void)
 }
 #endif
 
-ARC4RANDOM_EXPORT void
+/*defined in glibc 2,36
+ * ARC4RANDOM_EXPORT void
 arc4random_buf(void *buf_, size_t n)
 {
 	unsigned char *buf = buf_;
@@ -496,7 +497,7 @@ arc4random_buf(void *buf_, size_t n)
 		buf[n] = arc4_getbyte();
 	}
 	ARC4_UNLOCK_();
-}
+} */
 
 #ifndef ARC4RANDOM_NOUNIFORM
 /*
