diff --git a/meson.build b/meson.build
index 39b739e..fa48eb9 100644
--- a/meson.build
+++ b/meson.build
@@ -72,10 +72,6 @@ gobject_dep = dependency('gobject-2.0', version: glib_version_info)
 
 #  everything else
 libsoup_dep = dependency('libsoup-3.0', required: false)
-#    fallback to libsoup2
-if not libsoup_dep.found()
-  libsoup_dep = dependency('libsoup-2.4')
-endif
 libxml_dep = dependency('libxml-2.0', version: '>= 2.6.0')
 libxslt_dep = dependency('libxslt', version: '>= 1.0.0')
 
